### PR TITLE
Qualtrics

### PR DESCRIPTION
**Pull request by akrausesap**
Incorporation of Qualtrics connectors

### CODE REVIEW

**Comment by a-thaler** [comment]
Do not commit binaries: qualtrics-event-gw/qualtrics-event-gw

**Comment by akrausesap** [comment]
done

File: qualtrics-event-gw/kubernetes/registrationfile.json [hunk 1]
```diff
@@ -0,0 +1,348 @@
+{
```
Comment (a-thaler) [inline qualtrics-event-gw/kubernetes/registrationfile.json]: Please move that to the related connector bundle

File: qualtrics-event-gw/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# qualtrics-event-gw
+
```
Comment (a-thaler) [inline qualtrics-event-gw/README.md]: Please add a summary on what that component is about, main responsibilities

File: qualtrics-event-gw/kubernetes/package.yaml [hunk 3]
```diff
@@ -0,0 +1,168 @@
+apiVersion: gateway.kyma-project.io/v1alpha2
```
Comment (a-thaler) [inline qualtrics-event-gw/kubernetes/package.yaml]: Please re-think if that content should be in the connector bundle only or do you really want to have it here as well.

**Comment by a-thaler** [comment]
I aligned the topic and for now it is fine to add the qualtric specific connector code. Please check again what really should be in the bundle itself (all the deployment manifests and configuration files?) and what should be here as part of the source code.

File: qualtrics-event-gw/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+COPY go.sum go.sum
+
+
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o qualtrics-event-gw main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:latest
```
[the remark this answers is not in the document]
Comment (akrausesap) [reply qualtrics-event-gw/Dockerfile]: changed

File: qualtrics-webhook-registration/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+COPY go.sum go.sum
+
+
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o qualtrics-webhook-registration
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:latest
```
[the remark this answers is not in the document]
Comment (akrausesap) [reply qualtrics-webhook-registration/Dockerfile]: changed

File: qualtrics-event-gw/kubernetes/registrationfile.json (hunk 1 above)
Comment (akrausesap) [reply qualtrics-event-gw/kubernetes/registrationfile.json]: removed

File: qualtrics-event-gw/README.md (hunk 2 above)
Comment (akrausesap) [reply qualtrics-event-gw/README.md]: done

File: qualtrics-event-gw/kubernetes/package.yaml (hunk 3 above)
Comment (akrausesap) [reply qualtrics-event-gw/kubernetes/package.yaml]: removed

**Pull request merged**